lib/ansible/modules/cloud/openstack/os_user_role.py

```diff
@@ -151,7 +151,7 @@ def main():
                 module.fail_json(msg="User %s is not valid" % user)
             filters['user'] = u['id']
         if group:
-            g = cloud.get_group(group)
+            g = cloud.get_group(group, domain_id=filters['domain'])
```

Comment:
In which way is this a fix? You should add a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs-how-to) for this. And maybe it's better to do this in a separate PR, so this fix can be backported.

Reply:
You're right @felixfontein. 
Reverted the code to open a new PR.